src/Apps/MyCollection/Routes/EditArtwork/MyCollectionArtworkForm.tsx

```diff
@@ -253,7 +256,16 @@ export const MyCollectionArtworkForm: React.FC<MyCollectionArtworkFormProps> = (
                               <BackLink
                                 // @ts-ignore
                                 as={RouterLink}
-                                onClick={() => setShouldShowBackModal(true)}
+                                onClick={() => {
+                                  const formIsDirty = !isEqual(
+                                    initialValues,
+                                    values
+                                  )
+
+                                  return formIsDirty
+                                    ? setShouldShowBackModal(true)
```

Comment:
**suggestion:** I know this was done in a different PR, but I feel like we improve the naming here. Maybe something like `showLeaveWithoutSavingModal` or `showQuitWithoutSavingModal`